# Final Notebook

## Introduction: 

The way that we used our datasets to come to our solution started with crime. Since the higher values were proven to be places that were not worth overlooking the crime, we had to begin at the smaller values. Each notebook looks at a few neighborhoods based on crime, housing values, and population. First we find the extremes datasets. In population we found three neighborhoods with the highest population and three neighborhoods with the lowest population. Then looked at how crime and housing values were in those neighborhoods. In crime we found the cities with the most crime and the least amount of crime and then looked at population and housing values for those cities. In housing values we found the top three neighborhoods with the most houses worth one million or over and the top three neighborhoods with the most houses worth ten thousand or less. We began looking at the population in all of the neighborhoods with the most crime.

The datasets we used are population data, housing price data and crime data in Allegheny County. Our goal is to find the neighborhood with the best qualities from all the above datasets. 

Our first approach was to sort the data and choose the “extreme” values. The reason we chose to find the highest and lowest values for each dataset is so that we can compare the “best” and “worst” conditions for each sub-metric. To explain, when looking at crime data, a high crime rate is the worst-case scenario, and a low crime rate would be the best.

We decided to look at the neighborhoods with the most expensive housing, the least expensive housing as well as the most populated and least populated neighborhoods: 


In [46]:
import pandas as pd
population = pd.read_csv('total-population.csv')
sortedP = population.sort_values('Estimate; Total')
value = pd.read_csv('value.csv')
values = value.sort_values('Estimate; Total: - $1,000,000 or more', axis = 0, ascending=False)
crime = pd.read_csv('crime.csv')
crime2 = crime['INCIDENTNEIGHBORHOOD'].value_counts().rename_axis("Neighborhood")
total = len(crime)

## Our initial finding:

### Estimated Total Population: 309327 People

### The 3 most populated neighborhoods:

- Squirrel Hill South: 16042 People
- Shadyside: 13562 People
- Brookline: 13160 People 


In [7]:
display(sortedP.tail(3))

Neighborhood  Id  Estimate; Total  Margin of Error; Total
13            Brookline  14          13160.0              943.065215
68            Shadyside  69          13562.0              861.984919
77  Squirrel Hill South  78          16042.0              889.712313

### The 3 least populated neighborhoods:

- North Shore: 205 People
- South Shore: 12 People
- Chateau: 3 People

In [8]:
display(sortedP.head(3))

Neighborhood  Id  Estimate; Total  Margin of Error; Total
21      Chateau  22              3.0                3.000000
71  South Shore  72             12.0               14.000000
57  North Shore  58            205.0               66.753277

### Top 3 Towns With Highest Amount Houses Worth 1 Million or More
1. Squirrel Hill North
2. Shadyside
3. Mount Washington

In [12]:
display(values.head(3))

Neighborhood  Id  Estimate; Total:  Margin of Error; Total:  \
76  Squirrel Hill North  77            2065.0               157.178243   
68            Shadyside  69            2030.0               225.794154   
53     Mount Washington  54            1927.0               180.557470   

    Estimate; Total: - Less than $10,000  \
76                                  11.0   
68                                   0.0   
53                                  30.0   

    Margin of Error; Total: - Less than $10,000  \
76                                    34.985711   
68                                    37.416574   
53                                    32.511536   

    Estimate; Total: - $10,000 to $14,999  \
76                                    0.0   
68                                   11.0   
53                                   10.0   

    Margin of Error; Total: - $10,000 to $14,999  \
76                                     31.622777   
68                                     40.298883   
53                                     31.559468   

    Estimate; Total: - $15,000 to $19,999  \
76                                    0.0   
68                                    0.0   
53                                    8.0   

    Margin of Error; Total: - $15,000 to $19,999  ...  \
76                                     31.622777  ...   
68                                     37.416574  ...   
53                                     30.724583  ...   

    Estimate; Total: - $300,000 to $399,999  \
76                                    342.0   
68                                    327.0   
53                                     48.0   

    Margin of Error; Total: - $300,000 to $399,999  \
76                                       85.737973   
68                                      126.980313   
53                                       38.392708   

    Estimate; Total: - $400,000 to $499,999  \
76                                    336.0   
68                                    168.0   
53                                      9.0   

    Margin of Error; Total: - $400,000 to $499,999  \
76                                       85.527773   
68                                       68.403216   
53                                       31.559468   

    Estimate; Total: - $500,000 to $749,999  \
76                                    363.0   
68                                    309.0   
53                                     45.0   

    Margin of Error; Total: - $500,000 to $749,999  \
76                                       91.214034   
68                                      119.314710   
53                                       40.074930   

    Estimate; Total: - $750,000 to $999,999  \
76                                    250.0   
68                                     92.0   
53                                      0.0   

    Margin of Error; Total: - $750,000 to $999,999  \
76                                       76.576759   
68                                       59.270566   
53                                       30.000000   

    Estimate; Total: - $1,000,000 or more  \
76                                  136.0   
68                                   70.0   
53                                   27.0   

    Margin of Error; Total: - $1,000,000 or more  
76                                     56.000000  
68                                     50.813384  
53                                     34.073450  

[3 rows x 52 columns]

### Top 3 Towns With Highest Amount Houses Worth 10k or Less
1. Perry South
2. Brookline
3. Allentown

In [14]:
values2 = value.sort_values('Estimate; Total: - Less than $10,000', axis = 0, ascending=False)
display(values2.head(3))

Neighborhood  Id  Estimate; Total:  Margin of Error; Total:  \
62  Perry South  63             869.0               124.911969   
13    Brookline  14            4376.0               323.978394   
2     Allentown   3             575.0               124.004032   

    Estimate; Total: - Less than $10,000  \
62                                  83.0   
13                                  54.0   
2                                   48.0   

    Margin of Error; Total: - Less than $10,000  \
62                                    57.428216   
13                                    47.989582   
2                                     36.400549   

    Estimate; Total: - $10,000 to $14,999  \
62                                   22.0   
13                                   39.0   
2                                    19.0   

    Margin of Error; Total: - $10,000 to $14,999  \
62                                     29.189039   
13                                     47.095647   
2                                      21.840330   

    Estimate; Total: - $15,000 to $19,999  \
62                                   18.0   
13                                    5.0   
2                                    32.0   

    Margin of Error; Total: - $15,000 to $19,999  ...  \
62                                     29.291637  ...   
13                                     38.262253  ...   
2                                      33.166248  ...   

    Estimate; Total: - $300,000 to $399,999  \
62                                     21.0   
13                                     11.0   
2                                       0.0   

    Margin of Error; Total: - $300,000 to $399,999  \
62                                       35.057096   
13                                       41.521079   
2                                        17.320508   

    Estimate; Total: - $400,000 to $499,999  \
62                                      0.0   
13                                      0.0   
2                                       0.0   

    Margin of Error; Total: - $400,000 to $499,999  \
62                                       24.494897   
13                                       38.729833   
2                                        17.320508   

    Estimate; Total: - $500,000 to $749,999  \
62                                      0.0   
13                                      8.0   
2                                       5.0   

    Margin of Error; Total: - $500,000 to $749,999  \
62                                       24.494897   
13                                       41.097445   
2                                        16.248077   

    Estimate; Total: - $750,000 to $999,999  \
62                                      0.0   
13                                     22.0   
2                                       0.0   

    Margin of Error; Total: - $750,000 to $999,999  \
62                                       24.494897   
13                                       52.621288   
2                                        17.320508   

    Estimate; Total: - $1,000,000 or more  \
62                                    0.0   
13                                    8.0   
2                                     0.0   

    Margin of Error; Total: - $1,000,000 or more  
62                                     24.494897  
13                                     39.293765  
2                                      17.320508  

[3 rows x 52 columns]

### The 3 Neighborhoods with the most crime:

- Central Business District:    210 Cases
- South Side Flats:             122 Cases
- Carrick:                      118 Cases

In [18]:
display(crime2.head(3))

Neighborhood
Central Business District    210
South Side Flats             122
Carrick                      118
Name: INCIDENTNEIGHBORHOOD, dtype: int64

### The 7 Neighbourhoods with the least crime:

- Esplen:                         3 Cases
- Glen Hazel:                     2 Cases
- Regent Square:                  2 Cases
- Mount Oliver Borough:           2 Cases
- Outside County:                 2 Cases
- Summer Hill:                    2 Cases
- Ridgemont:                      1 Cases

In [19]:
display(crime2.tail(7))

Neighborhood
Esplen             3
Glen Hazel         2
Regent Square      2
Mt. Oliver Boro    2
Outside County     2
Summer Hill        2
Ridgemont          1
Name: INCIDENTNEIGHBORHOOD, dtype: int64

## QUALITIES OF NEIGHBORHOODS OF HIGHEST POPULATION:

Let's discuss population. We wanted a population that was around half of the second highest population because Squirrel Hill South was sort of an outlier as the highest. We wanted a population around 3000-6000.

First, lets look at Squirrel Hill South. Squirrel Hill South has the highest population at 16042 which is almost 3000 more than the second runner up for population, Shadyside. There were 51 crime cases in the past 30 days. Most of the houses in Squirrel Hill South are valued at over 200k with a tie with Mount Washington in third for the amount of houses 1 million or over at 27. The population is extremely large and the amount of expensive housing in this area likely makes getting a house here difficult.
We cannot justify choosing Squirrel Hill South as the best neighborhood. 

Next, we’ll look at Shadyside. The population is high at 13562 people. It is one of the most populated neighborhoods. Shadyside has 76 houses worth 1 million or more out of their 2030 total houses. Most of the houses in Shadyside are above 200k which is still very pricy for most people. Shadyside has 54 crimes in the last 30 days. It’s a higher crime rate which isn’t safe. We cannot justify choosing Shadyside as the best neighborhood.

The third most popular neighborhood is Brookline at 13160 people. Brookline was actually in second place for the most houses valued at 10k or less with 54 houses. This doesn’t mean much since Brookline has 4376 houses, but most of Brookline’s houses are actually in good price ranges below 200k. Brookline had 76 crime cases. While Brookline isn’t terrible there is definitely a neighborhood that would be safer with a less overwhelming population.
We cannot justify choosing Brookline as the best neighborhood.

In [61]:
array = ['Squirrel Hill South', 'Shadyside', 'Brookline']
display(value.loc[value['Neighborhood'].isin(array)])

Neighborhood  Id  Estimate; Total:  Margin of Error; Total:  \
13            Brookline  14            4376.0               323.978394   
68            Shadyside  69            2030.0               225.794154   
77  Squirrel Hill South  78            3069.0               245.104060   

    Estimate; Total: - Less than $10,000  \
13                                  54.0   
68                                   0.0   
77                                  38.0   

    Margin of Error; Total: - Less than $10,000  \
13                                    47.989582   
68                                    37.416574   
77                                    47.916594   

    Estimate; Total: - $10,000 to $14,999  \
13                                   39.0   
68                                   11.0   
77                                    0.0   

    Margin of Error; Total: - $10,000 to $14,999  \
13                                     47.095647   
68                                     40.298883   
77                                     36.055513   

    Estimate; Total: - $15,000 to $19,999  \
13                                    5.0   
68                                    0.0   
77                                   32.0   

    Margin of Error; Total: - $15,000 to $19,999  ...  \
13                                     38.262253  ...   
68                                     37.416574  ...   
77                                     50.803543  ...   

    Estimate; Total: - $300,000 to $399,999  \
13                                     11.0   
68                                    327.0   
77                                    593.0   

    Margin of Error; Total: - $300,000 to $399,999  \
13                                       41.521079   
68                                      126.980313   
77                                      143.763695   

    Estimate; Total: - $400,000 to $499,999  \
13                                      0.0   
68                                    168.0   
77                                    257.0   

    Margin of Error; Total: - $400,000 to $499,999  \
13                                       38.729833   
68                                       68.403216   
77                                       87.177979   

    Estimate; Total: - $500,000 to $749,999  \
13                                      8.0   
68                                    309.0   
77                                    338.0   

    Margin of Error; Total: - $500,000 to $749,999  \
13                                       41.097445   
68                                      119.314710   
77                                      108.912809   

    Estimate; Total: - $750,000 to $999,999  \
13                                     22.0   
68                                     92.0   
77                                    149.0   

    Margin of Error; Total: - $750,000 to $999,999  \
13                                       52.621288   
68                                       59.270566   
77                                      100.084964   

    Estimate; Total: - $1,000,000 or more  \
13                                    8.0   
68                                   70.0   
77                                   27.0   

    Margin of Error; Total: - $1,000,000 or more  
13                                     39.293765  
68                                     50.813384  
77                                     46.957428  

[3 rows x 52 columns]

In [48]:
shlcr = 51/total *100 
ssdcr = 54/total *100
brocr = 76/total *100

print ("Squirrel Hill South Crime Rate:", shlcr, "%")
print ("Shadyside Crime Rate:", ssdcr, "%")
print ("Brookline Crime Rate:", brocr, "%")

Squirrel Hill South Crime Rate: 1.7513736263736264 %
Shadyside Crime Rate: 1.8543956043956045 %
Brookline Crime Rate: 2.60989010989011 %


In [56]:
array = ['Squirrel Hill South', 'Shadyside', 'Brookline']
display(population.loc[population['Neighborhood'].isin(array)])

Neighborhood  Id  Estimate; Total  Margin of Error; Total
13            Brookline  14          13160.0              943.065215
68            Shadyside  69          13562.0              861.984919
77  Squirrel Hill South  78          16042.0              889.712313

## QUALITIES OF NEIGHBORHOODS WITH HIGHEST HOUSING RATES:

Now lets discuss the neighborhoods with the most expensive houses to see if there any redeeming qualities to justify them. We decided to look for housing values within 50k-150k for people who are less fortunate with money.

First, we’ll take a look at Squirrel Hill North. Squirrel Hill North has 136 houses worth 1 million or more out of their 2065 total houses. Squirrel Hill North has very little houses worth anything below 300k. They’re extremely expensive. Squirrel Hill North had 39 crimes which when compared to the “extreme” crime rate neighborhoods isn’t too bad, it still isn’t as safe as others. The population of Squirrel Hill North is also very high at 11336 people. 
We cannot justify choosing Squirrel Hill North as the best neighborhood. 

Shadyside has the second most houses worth 1 million or more at 76, but we’ve already deemed it unjustifiable to choose Shadyside as the best neighborhood.

Finally, we’ll look at Mount Washington. Mount Washington has 37 houses valued at over 1 million or more out of their total 1927 houses. A good portion of Mount Washington’s houses are actually in the lower price ranges, but Mount Washington has a higher crime rate than both Shadyside and Squirrel Hill North at 72. The population is lower at 8743 people, but still a little high. The crime rate in Mount Washington makes the neighborhood appear unsafe.
We cannot justify Mount Washington as the best neighborhood.

In [58]:
array2 = ['Squirrel Hill North', 'Shadyside', 'Mount Washington']
display(value.loc[value['Neighborhood'].isin(array2)])

Neighborhood  Id  Estimate; Total:  Margin of Error; Total:  \
53     Mount Washington  54            1927.0               180.557470   
68            Shadyside  69            2030.0               225.794154   
76  Squirrel Hill North  77            2065.0               157.178243   

    Estimate; Total: - Less than $10,000  \
53                                  30.0   
68                                   0.0   
76                                  11.0   

    Margin of Error; Total: - Less than $10,000  \
53                                    32.511536   
68                                    37.416574   
76                                    34.985711   

    Estimate; Total: - $10,000 to $14,999  \
53                                   10.0   
68                                   11.0   
76                                    0.0   

    Margin of Error; Total: - $10,000 to $14,999  \
53                                     31.559468   
68                                     40.298883   
76                                     31.622777   

    Estimate; Total: - $15,000 to $19,999  \
53                                    8.0   
68                                    0.0   
76                                    0.0   

    Margin of Error; Total: - $15,000 to $19,999  ...  \
53                                     30.724583  ...   
68                                     37.416574  ...   
76                                     31.622777  ...   

    Estimate; Total: - $300,000 to $399,999  \
53                                     48.0   
68                                    327.0   
76                                    342.0   

    Margin of Error; Total: - $300,000 to $399,999  \
53                                       38.392708   
68                                      126.980313   
76                                       85.737973   

    Estimate; Total: - $400,000 to $499,999  \
53                                      9.0   
68                                    168.0   
76                                    336.0   

    Margin of Error; Total: - $400,000 to $499,999  \
53                                       31.559468   
68                                       68.403216   
76                                       85.527773   

    Estimate; Total: - $500,000 to $749,999  \
53                                     45.0   
68                                    309.0   
76                                    363.0   

    Margin of Error; Total: - $500,000 to $749,999  \
53                                       40.074930   
68                                      119.314710   
76                                       91.214034   

    Estimate; Total: - $750,000 to $999,999  \
53                                      0.0   
68                                     92.0   
76                                    250.0   

    Margin of Error; Total: - $750,000 to $999,999  \
53                                       30.000000   
68                                       59.270566   
76                                       76.576759   

    Estimate; Total: - $1,000,000 or more  \
53                                   27.0   
68                                   70.0   
76                                  136.0   

    Margin of Error; Total: - $1,000,000 or more  
53                                     34.073450  
68                                     50.813384  
76                                     56.000000  

[3 rows x 52 columns]

In [59]:
shncr = 39/total *100 
ssdcr = 54/total *100
mwncr = 72/total *100

print ("Squirrel Hill North Crime Rate:", shncr, "%")
print ("Shadyside Crime Rate:", ssdcr, "%")
print ("Mount Washington Crime Rate:", mwncr, "%")

Squirrel Hill North Crime Rate: 1.3392857142857142 %
Shadyside Crime Rate: 1.8543956043956045 %
Mount Washington Crime Rate: 2.4725274725274726 %


In [60]:
display(population.loc[population['Neighborhood'].isin(array2)])

Neighborhood  Id  Estimate; Total  Margin of Error; Total
53     Mount Washington  54           8743.0              613.158218
68            Shadyside  69          13562.0              861.984919
76  Squirrel Hill North  77          11336.0              767.783173

## QUALITIES OF NEIGHBORHOODS WITH HIGHEST CRIME RATES:

Let’s discuss the three neighborhoods with the highest crime and see if there are any redeeming qualities that would justify us choosing them as the “best”.

First let’s look at Central Business District. The estimated population is 4060 people, and the housing prices tend to skew more towards prices of 100 thousand and higher. The population is within our defined threshold of 3000-6000 people however the housing is expensive compared to other neighborhoods. 
We cannot justify choosing Central Business District as the best neighborhood

Next let’s look at South Side Flats. The estimated population is 6292 people, and the housing prices also skew to more expensive pricing. Namely, 70 thousand – 400 thousand. The population is slightly above the threshold and the housing price is generally higher than most neighborhoods.
We cannot justify choosing South Side Flats as the best neighborhood

Finally, Carrick. The estimated population is 10122 people, and the majority of the housing prices skew towards cheaper housing ranging from roughly 100 thousand and lower. Where a large majority are ranging from 10 thousand to 70 thousand. The population is much higher than our threshold and the housing prices are generally too low meaning that the quality of the housing isn’t great.
We cannot justify choosing Carrick as the best neighborhood


In [66]:
array3 = ['Central Business District', 'South Side Flats', 'Carrick']
display(value.loc[value['Neighborhood'].isin(array3)])

Neighborhood  Id  Estimate; Total:  Margin of Error; Total:  \
15                    Carrick  16            2660.0               223.861564   
16  Central Business District  17             402.0               103.344085   
72           South Side Flats  73            1413.0               209.234318   

    Estimate; Total: - Less than $10,000  \
15                                   9.0   
16                                   0.0   
72                                  40.0   

    Margin of Error; Total: - Less than $10,000  \
15                                    35.284558   
16                                    24.494897   
72                                    55.686623   

    Estimate; Total: - $10,000 to $14,999  \
15                                   36.0   
16                                    0.0   
72                                   10.0   

    Margin of Error; Total: - $10,000 to $14,999  \
15                                     40.236799   
16                                     24.494897   
72                                     27.495454   

    Estimate; Total: - $15,000 to $19,999  \
15                                   42.0   
16                                    0.0   
72                                    0.0   

    Margin of Error; Total: - $15,000 to $19,999  ...  \
15                                     62.425956  ...   
16                                     24.494897  ...   
72                                     24.494897  ...   

    Estimate; Total: - $300,000 to $399,999  \
15                                     15.0   
16                                     11.0   
72                                    159.0   

    Margin of Error; Total: - $300,000 to $399,999  \
15                                       42.720019   
16                                       25.159491   
72                                       65.115282   

    Estimate; Total: - $400,000 to $499,999  \
15                                      0.0   
16                                     37.0   
72                                     52.0   

    Margin of Error; Total: - $400,000 to $499,999  \
15                                       36.055513   
16                                       39.420807   
72                                       44.564560   

    Estimate; Total: - $500,000 to $749,999  \
15                                      7.0   
16                                     83.0   
72                                     20.0   

    Margin of Error; Total: - $500,000 to $749,999  \
15                                       36.660606   
16                                       70.887234   
72                                       31.559468   

    Estimate; Total: - $750,000 to $999,999  \
15                                      0.0   
16                                     10.0   
72                                     13.0   

    Margin of Error; Total: - $750,000 to $999,999  \
15                                       36.055513   
16                                       28.089144   
72                                       30.675723   

    Estimate; Total: - $1,000,000 or more  \
15                                    0.0   
16                                   10.0   
72                                    0.0   

    Margin of Error; Total: - $1,000,000 or more  
15                                     36.055513  
16                                     26.925824  
72                                     24.494897  

[3 rows x 52 columns]

In [63]:
CBDcr = 210/total *100 
SSFcr = 122/total *100
CARcr = 118/total *100

print ("Central Business District Crime Rate:", CBDcr, "%")
print ("South Side Flats Crime Rate:", SSFcr, "%")
print ("Carrick Crime Rate:", CARcr, "%")

Central Business District Crime Rate: 7.211538461538461 %
South Side Flats Crime Rate: 4.189560439560439 %
Carrick Crime Rate: 4.052197802197802 %


In [67]:
display(population.loc[population['Neighborhood'].isin(array3)])

Neighborhood  Id  Estimate; Total  Margin of Error; Total
15                    Carrick  16          10122.0              826.666196
16  Central Business District  17           4060.0              458.026200
72           South Side Flats  73           6292.0              747.592135

## Conclusion: 

A neighborhood with too many people would be too crowded to live in, but you also don’t want to live in a neighborhood that has too few people. So it would be best to find the city with the least crime that has a good population middleground (as defined previously). Mount Oliver Borough was perfect with only two crime cases in the past thirty days and a population of 3399 people. The only thing left to do was make sure it didn’t have extreme housing values. A good neighborhood must be affordable, but not too affordable. You don’t want to live in a neighborhood that has houses that are worn down. In Mount Oliver Borough, the interval 40k-49.9k had the most houses at around 168 houses. That may seem a little low compared to other areas, but when you take a deeper look into Mount Oliver Borough’s housing values there are about 96 in the 50k-59.k interval and 117 in the 60k-69.9k total. As you get into higher values there are less options, but Mount Oliver Borough is a safe neighborhood with low population and affordable housing that isn’t too low that the houses would be substandard.

In [70]:
display(value.loc[value['Neighborhood'] == "Mount Oliver Borough"])
display(population.loc[population['Neighborhood'] == "Mount Oliver Borough"])

Neighborhood  Id  Estimate; Total:  Margin of Error; Total:  \
52  Mount Oliver Borough  53             595.0               118.692881   

    Estimate; Total: - Less than $10,000  \
52                                   7.0   

    Margin of Error; Total: - Less than $10,000  \
52                                    21.071308   

    Estimate; Total: - $10,000 to $14,999  \
52                                    0.0   

    Margin of Error; Total: - $10,000 to $14,999  \
52                                          20.0   

    Estimate; Total: - $15,000 to $19,999  \
52                                    0.0   

    Margin of Error; Total: - $15,000 to $19,999  ...  \
52                                          20.0  ...   

    Estimate; Total: - $300,000 to $399,999  \
52                                      0.0   

    Margin of Error; Total: - $300,000 to $399,999  \
52                                            20.0   

    Estimate; Total: - $400,000 to $499,999  \
52                                      0.0   

    Margin of Error; Total: - $400,000 to $499,999  \
52                                            20.0   

    Estimate; Total: - $500,000 to $749,999  \
52                                      0.0   

    Margin of Error; Total: - $500,000 to $749,999  \
52                                            20.0   

    Estimate; Total: - $750,000 to $999,999  \
52                                      0.0   

    Margin of Error; Total: - $750,000 to $999,999  \
52                                            20.0   

    Estimate; Total: - $1,000,000 or more  \
52                                    0.0   

    Margin of Error; Total: - $1,000,000 or more  
52                                          20.0  

[1 rows x 52 columns]

Neighborhood  Id  Estimate; Total  Margin of Error; Total
52  Mount Oliver Borough  53           3399.0              462.961121

In [71]:
MOB = 2/total *100
print ("Mount Oliver Borough Crime Rate:", MOB, "%")

Mount Oliver Borough Crime Rate: 0.06868131868131869 %


## Reflection:

Sharif: The final result that we got may seem strange considering that I'm not American and only ever lived on Oakland. Mount Oliver Borough sounds like a stark contrast compared to Oakland but it also sounds like a relatively safe and cozy place to live in. I've been living in Oakland for nearly a year now and I've been loving it. However, I can't see myself living here long-term and I would prefer a safer and cozier place, such as Mount Oliver Borough, in the future. 

Darian: I've actually lived in the Pittsburgh area my entire life, but I haven't been to Mount Oliver Borough. I'd honestly never even heard of it. It seems like a comfy area, but it's hard to change my mind about where I grew up. I grew up in Carnegie which didn't sh